### ISSUE TITLE
TMO Speed Up Map causes NullReferenceException in property Dependencies

### ISSUE THREAD

**Issue by ozontm** [stefan-baumann/ManiaPlanetSharp#30]
Missing `<dep></dep>` causes parsing of the file to fail.

[Rollercoaster.Challenge.zip](https://github.com/stefan-baumann/ManiaPlanetSharp/files/8986356/Rollercoaster.Challenge.zip)

**Comment by stefan-baumann** [comment]
I can't test this this right now, but does this actually cause any exceptions inside MP# or just in your downstream code? Usually, the convention with MP# is that information that is not present in the file is actually set to `null`, which seems to be consistent in this case.

**Comment by ozontm** [comment]
Yep, that was a right call @stefan-baumann. The exception for the Dependencies getter mistakenly pointed me to the MetaDataProvider, also when I tested it using MP#View.
Issue can be closed, I'm now handling the null ref correctly on my end.

**Comment by stefan-baumann** [comment]
Apparently GameBoxView still has a problem with this, erroring out. That should be fixed, so I'm reopening this issue.
